CONTRIBUTING.md

```diff
@@ -53,10 +53,10 @@ For package version upgrades and such a one-line commit message is usually suffi
 Follow these steps to backport a change into a release branch in compliance with the [commit policy](https://nixos.org/nixpkgs/manual/#submitting-changes-stable-release-branches).
 
 1. Take note of the commits in which the change was introduced into `master` branch.
-2. Check out the target _release branch_, e.g. `release-20.09`. Do not use a _channel branch_ like `nixos-20.09` or `nixpkgs-20.09`.
```

Comment:
Wait, do we have nixpkgs-20.09 branch or nixpkgs-21.11 branch?

Maybe I should change this to nixpkgs-21.11-darwin?

Reply:
`
2. Check out the target _release branch_, e.g. `release-20.09`. Do not use a _channel branch_ like `nixos-20.09`.
`
It's good to tell people what to target and to avoid `nixos-20.09`

Reply:
Yeah I kept that, any suggestions with the current diff?